opm/common/utility/numeric/RootFinders.hpp

```diff
@@ -124,7 +124,7 @@ namespace Opm
             iterations_used = 0;
             // In every iteraton, x1 is the last point computed,
             // and x0 is the last point computed that makes it a bracket.
-            while (fabs(x1 - x0) >= 1e-9*eps) {
+            while (fabs(x1 - x0) >= eps) {
```

Comment:
Seems like quite a drastic change in tolerance.

Reply:
The original tolerance was so normally 1e-19 which mean that it was never trigged...

Reply:
Note that the RegulaFalsiBisection is used by the well model for thp computations. Maybe the change of the tolerance here also benefits the well models?